### DISCUSSION TITLE
Unwanted Search Results in User Account Without Adult Library Access

### DISCUSSION THREAD

**Discussion by cmcooper1980** [DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult#221] [Q&A] [answered]
So I didn't want to post this in issues as I am not sure if it is an issue...however, my particular problem is that Jellyfin is available to several users with only a select few with access to the Adult Library and if you do a search (Barbie for instance) in any of the user accounts WITHOUT Adult Library access the adult performers with the name Barbie or similar show up at the bottom of the search. You can see how this would be an issue when the actors/actresses profile picture is of them nude or in an act... Not sure if this is a function of the plugin that can be dealt with or Jellyfin itself...

**Comment by DirtyRacer1337** [comment, marked as answer]
Jellyfin problem, only fix I can do, https://github.com/DirtyRacer1337/Jellyfin.Plugin.PhoenixAdult/issues/19

**Reply by cmcooper1980** [reply]
OK thanks, just to be clear, do I just need to toggle "Disable Actors Saving to Metadata" and run the "Refresh People" task in Jellyfin or is it a little more involved?

**Reply by DirtyRacer1337** [reply]
yep but you should run task `Cleanup Actors` and they all would be deleted
